pallets/chainbridge/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+authors = ["Webb Technologies Inc."]
+description = "FRAME pallet for chain-bridge."
+edition = "2018"
+homepage = "https://substrate.dev"
+license = "Unlicense"
+name = "pallet-chainbridge"
+repository = "https://github.com/substrate-developer-hub/substrate-node-template/"
+version = "3.0.0"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+# alias "parity-scale-code" to "codec"
+[dependencies.codec]
+default-features = false
+features = ["derive"]
+package = "parity-scale-codec"
+version = "2.0.0"
+
+[dependencies]
+# third-party dependencies
+serde = { version = "1.0.101", optional = true }
+chainbridge = { version = "0.0.2", git = "https://github.com/ChainSafe/chainbridge-substrate.git", rev = "3999027", default-features = false}
```

Comment:
we don't want to depend on their pallet at all. I suggest copying it entirely over into its own pallet. We will need to modify it enough that this makes sense.

Reply:
Yeah, I also feel so. I will copy it in a `chainbridge` pallet then.